src/components/CardProgressSlider.vue

```diff
@@ -0,0 +1,229 @@
+          >
+            <div class="relative-position">
+              <!-- Progress bar -->
+              <q-circular-progress
+                rounded
+                class="text-white"
+                :value="card.progress"
+                :size="circleSize"
+                :thickness="0.05"
+                color="white"
+                track-color="blue-grey-10"
+                data-cy="card-progress-circular"
+              />
+              <!-- Progress label -->
+              <div class="text-white absolute-center text-center">
+                <!-- Caption -->
+                <div class="text-caption">
+                  {{ $t('index.cardProgressSlider.toDate') }}
+                </div>
+                <!-- Number -->
+                <div class="circular-progress-number q-mt-xs">
+                  {{ card.progress }}%
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <!-- Section stats -->
+          <div class="col flex column gap-16 gt-xs q-pl-xl">
+            <div
+              v-for="stat in card.stats"
+              :key="stat.title"
+              data-cy="card-progress-stats"
+            >
+              <!-- Title stats -->
+              <div class="stats-title text-uppercase text-caption">
+                {{ stat.title }}
+              </div>
+              <!-- List stats -->
+              <q-list dark dense class="q-mt-lg">
+                <q-item
+                  v-for="item in stat.items"
+                  :key="item.id"
+                  class="stats-value"
+                  style="padding: 0"
+                >
+                  {{ item.text }}
+                </q-item>
+              </q-list>
+            </div>
+          </div>
+        </div>
+      </q-card-section>
+
+      <q-separator color="blue-grey-8 lt-sm" />
+
+      <!-- Card footer (mobile) -->
+      <q-card-section
+        class="card-image-section-content lt-sm"
+        data-cy="card-progress-footer-mobile"
+      >
+        <!-- Timeline showing the progress of the challenge -->
+        <div class="min-w-180" data-cy="card-progress-timeline">
+          <!-- Timeline label -->
+          <div class="text-subtitle2 text-center">
+            {{ card.duration?.current }} / {{ card.duration?.total }}
+            {{ $t('index.cardProgressSlider.timeline') }}
+          </div>
+          <!-- Timeline progress bar -->
+          <q-linear-progress
+            class="q-mt-xs"
+            :value="timelineValue"
+            color="white"
+            rounded
+          />
+        </div>
+      </q-card-section>
+    </q-img>
+  </q-card>
+</template>
+
+<style scoped lang="scss">
+:deep(.q-list--dense > .q-item) {
+  min-height: 24px;
+}
+:deep(.q-img > div) {
+  padding-bottom: 0 !important;
+}
+:deep(.q-img > *) {
+  position: relative !important;
+}
+:deep(.q-img > * > .card-image-section-content) {
+  position: relative !important;
+}
+.circular-progress-number {
```

Comment:
Same CSS class declaration as in the `CardProgress` component. It would be good when this CSS class declaration be global (plus add comment wich compoenents use this CSS class, please).

Reply:
Refactored into circular-progress.scss